docs/upload_mapping_file.md

```diff
@@ -1,71 +1,91 @@
+Error Tracking processes errors collected from the RUM Android SDK. 
 
-If your mobile Android source code is obfuscated, upload your Proguard/R8 mapping file to Datadog so that your different stack traces can be deobfuscated. For a given error, you can get access to the file path, the line number, as well as a code snippet for each frame of the related stack trace.
+Enable Android Crash Reporting and Error Tracking to get comprehensive crash reports and error trends with Real User Monitoring. With this feature, you can access:
+
+- Aggregated Android crash dashboards and attributes
+- Deobfuscated Android crash reports
+- Trend analysis with Android error tracking
+
+Your crash reports appear in [**Error Tracking**][1].
+
+## Setup
+
+If you have not set up the Android SDK yet, follow the [in-app setup instructions][2] or see the [Android RUM setup documentation][3].
+
+1. Download the latest version of the [RUM Android SDK][4].
+2. Configure your application's `version`, `env`, and `service` when [initializing the SDK][5].
+3. Upload your Proguard/R8 mapping file to Datadog to access deobfuscated stack traces. 
+
+For any given error, you can access to the file path, line number, as well as a code snippet for each frame of the related stack trace.
 
 ## Upload your mapping file
 
 {{< tabs >}}
 {{% tab "US" %}}
 
-1. Add the [Gradle Plugin][1] to your Gradle project, using the following snippet.
+1. Add the [Android Gradle Plugin][1] to your Gradle project using the following code snippet.
 
-```groovy
-// In your app's build.gradle script
-plugins {
-    id("com.datadoghq.dd-sdk-android-gradle-plugin") version "x.y.z"
-}
-```
+   ```groovy
+   // In your app's build.gradle script
+   plugins {
+       id("com.datadoghq.dd-sdk-android-gradle-plugin") version "x.y.z"
+   }
+   ```
 
-2. [Create a new and dedicated Datadog API key][2] and export it as an environment variable named `DD_API_KEY` (alternatively you can pass it as a task property).
-3. (Optional) Configure the plugin to upload files to the EU region by exporting two additional environment variables: `export DATADOG_SITE="datadoghq.eu"` and `export DATADOG_API_HOST="api.datadoghq.eu"`.
-4. Run the upload task after your obfuscated APK has been built:
-```bash
-./gradlew uploadMappingRelease
-```
-**Note**: If your project uses additional flavors, the plugin provides an upload task for each variant with obfuscation enabled. In this case, initialize SDK with a proper variant name (the necessary API is available since version `1.8.0`).
+2. [Create a dedicated Datadog API key][2] and export it as an environment variable named `DD_API_KEY` (alternatively, pass it as a task property).
+3. Optionally, configure the plugin to upload files to the EU region by exporting two additional environment variables: `export DATADOG_SITE="datadoghq.eu"` and `export DATADOG_API_HOST="api.datadoghq.eu"`.
```

Comment:
I must have missed when those lines were added, but we don't support those two environment variables in the Gradle plugin (that must be a copy-paste from the `datadog-ci` tool). 


```suggestion
3. Optionally, configure the plugin to upload files to the EU region by configuring the plugin in your Gradle script: 
    datadog {
        site = "EU1"
    }
```